heat-templates/openstack-network.yml

```diff
@@ -0,0 +1,58 @@
+heat_template_version: 2015-04-30
+
+description: >
+  Template to generate the socok8s network including a router
+  to access the external network
+
+parameters:
+  prefix:
+    type: string
+    label: Stack name
+    description: Prefix for the resource names
+  network_external:
+    type: string
+    label: Public network name
+    description: The name of the public network
+    default: floating
+
```

Comment:
I would prefer if this would be creating network_external, network, subnet directly within the parameters, and in ansible be smart and use the prefix if necessary.

Reply:
Is this still relevant? If yes, I think external networks usually can only be created by admin users and need to exist before. So we can create that with the stack.

Reply:
@rhafer I guess you mean we **can not** (instead of can) in case of the external network.
The parameter `network_external` is there to tell the router the external gateway info.

Reply:
I think it's fine.

Reply:
> @rhafer I guess you mean we **can not** (instead of can) in case of the external network?

Ooops, yes.